### PR TITLE
appsso: add LDAP identity provider configuration

### PR DESCRIPTION
**Pull request by Kehrlann**
Only for TAP 1.4

### CODE REVIEW

File: app-sso/service-operators/identity-providers.md [hunk 1]
```diff
@@ -103,24 +101,21 @@ spec:
...
+          searchFilter: cn={0}
         group:
-          searchFilter: member={0}
-          searchBase: ou=Users,o=5d03d6ac6eed091436a8d664,dc=jumpcloud,dc=com
-          searchSubTree: true
-          searchDepth: 10
-          roleAttribute: cn
+          search:
+            filter: member={0}
+            base: ou=Users,dc=example,dc=com
```
Comment (candicenonsense) [inline app-sso/service-operators/identity-providers.md:115]: @Kehrlann do we want to refer to this as memberOf to make it look more AD-ish?

Comment (Kehrlann) [reply app-sso/service-operators/identity-providers.md:115]: @candicenonsense Ah sorry, I missed this comment somehow.

`memberOf` is only used when `group.search` is turned off. For clarification, `memberOf` is an attribute of the user object, while `member` is an attribute of the group object.

**Comment by YugangW** [comment]
Hi @Kehrlann, Groups to roles mapping seems like an empty section: https://github.com/pivotal/docs-tap/pull/1941/files#diff-1dbf534060853f91e91f0aaee6b44100649a80bc6900f89c07aece293b869b4aR145

Is this something by design? Thanks!

**Comment by Kehrlann** [comment]
@YugangW good catch, that's actually an issue. Feel free to remove it entirely.

**Pull request merged**